edxsearch/__init__.py

```diff
@@ -1,3 +1,3 @@
 """ Container module for testing / demoing search """
 
-__version__ = '4.0.0'
+__version__ = '3.2.0'
```

Comment:
why are we downgrading this version

Reply:
@awais786 had mistakenly bumped the major version in the cleanup PR. To maintain consistency in the version I am downgrading that in this PR before we release it to PyPi. 
p.s Version 4.0 was never released and didn't make it to PyPI